src/mtx/commands/uplink/GetHalfHoursResponse.ts

```diff
@@ -0,0 +1,338 @@
+            },
+            periods: [
+                {tariff: 1, energy: 1111},
+                {tariff: 1, energy: 1222},
+                {tariff: 1, energy: 1333},
+                {tariff: 1, energy: 1444},
+                {tariff: 1, energy: 1555},
+                {tariff: 1, energy: 1666},
+                {tariff: 1, energy: 1777},
+                {tariff: 1, energy: 1888},
+                {tariff: 1, energy: 1999},
+                {tariff: 1, energy: 2000},
+                {tariff: 1, energy: 2111},
+                {tariff: 1, energy: 2222},
+                {tariff: 1, energy: 2333},
+                {tariff: 1, energy: 2444},
+                {tariff: 1, energy: 2555},
+                {tariff: 1, energy: 2666},
+                {tariff: 1, energy: 2777},
+                {tariff: 1, energy: 2888},
+                {tariff: 1, energy: 2999},
+                {tariff: 1, energy: 3000},
+                {tariff: 1, energy: 3111},
+                {tariff: 1, energy: 3222},
+                {tariff: 1, energy: 3333},
+                {tariff: 1, energy: 3444},
+                {tariff: 1, energy: 3555},
+                {tariff: 1, energy: 3666},
+                {tariff: 1, energy: 3777},
+                {tariff: 1, energy: 3888},
+                {tariff: 1, energy: 3999},
+                {tariff: 1, energy: 4000},
+                {tariff: 1, energy: 4111},
+                {tariff: 1, energy: 4222},
+                {tariff: 1, energy: 4333},
+                {tariff: 1, energy: 4444},
+                {tariff: 1, energy: 4555},
+                {tariff: 1, energy: 4666},
+                {tariff: 1, energy: 4777},
+                {tariff: 1, energy: 4888},
+                {tariff: 1, energy: 4999},
+                {tariff: 1, energy: 5000},
+                {tariff: 1, energy: 5222},
+                {tariff: 1, energy: 5333},
+                {tariff: 1, energy: 5444},
+                {tariff: 1, energy: 5555},
+                {tariff: 1, energy: 5666},
+                {tariff: 1, energy: 5777},
+                {tariff: 1, energy: 5888},
+                {tariff: 1, energy: 5999}
+            ]
+        },
+        hex: {
+            header: '15 63',
+            body: `18 03 16 44 57 44 c6 45 35 45 a4 46 13 46 82 46 f1 47 60 47 cf
+                   47 d0 48 3f 48 ae 49 1d 49 8c 49 fb 4a 6a 4a d9 4b 48 4b b7 4b
+                   b8 4c 27 4c 96 4d 05 4d 74 4d e3 4e 52 4e c1 4f 30 4f 9f 4f a0
+                   50 0f 50 7e 50 ed 51 5c 51 cb 52 3a 52 a9 53 18 53 87 53 88 54
+                   66 54 d5 55 44 55 b3 56 22 56 91 57 00 57 6f`
+        }
+    },
+    {
+        name: 'empty data',
+        parameters: {
+            date: {
+                year: 22,
+                month: 6,
+                date: 18
+            },
+            periods: []
+        },
+        hex: {
+            header: '15 63',
+            body: `16 06 12 ff ff 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
```

Comment:
`ff ff` means there is no data at that time. in the command will be no `00 00`

Reply:
fixed